### PR TITLE
Feature/footer stuff

### PR DESCRIPTION
**Pull request by robineng**

### CODE REVIEW

File: src/Atoms/Icon/components/SocialYouTube.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      <g stroke="none" fillRule="nonzero">
+        <path
+          d="M15.8403556,3.37304308 C15.8403556,3.37304308 16,4.6386313 16,5.90421954 L16,7.09091484 C16,8.35632926 15.8403556,9.6219175 15.8403556,9.6219175 C15.8403556,9.6219175 15.6839111,10.6996572 15.2040889,11.1744048 C14.5959111,11.7973807 13.9137778,11.8003349 13.6010667,11.8368272 C11.3621333,11.9949606 8,12 8,12 C8,12 3.84017779,11.9628126 2.56,11.8427355 C2.20391111,11.7773969 1.40444444,11.7973807 0.795911112,11.1744048 C0.31608889,10.6996572 0.16,9.6219175 0.16,9.6219175 C0.16,9.6219175 0,8.35632926 0,7.09091484 L0,5.90421954 C0,4.6386313 0.16,3.37304308 0.16,3.37304308 C0.16,3.37304308 0.31608889,2.29530339 0.795911112,1.82055576 C1.40444444,1.19775359 2.08622222,1.19479945 2.39893333,1.15830714 C4.63804446,1 7.99644443,1 7.99644443,1 L8.00355556,1 C8.00355556,1 11.3621333,1 13.6010667,1.15830714 C13.9137778,1.19479945 14.5959111,1.19775359 15.2040889,1.82055576 C15.6839111,2.29530339 15.8403556,3.37304308 15.8403556,3.37304308 Z M6.34826668,8.52853826 L10.6707556,6.33899937 L6.34755556,4.13434227 L6.34826668,8.52853826 Z"
+          id="Fill-195"
```
Comment (shmidt-i) [inline src/Atoms/Icon/components/SocialYouTube.tsx]: svgo missed this id?

File: src/Atoms/Separator/Separator.tsx [hunk 2]
```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import styled from 'styled-components';
 import { SeparatorComponent, StyledProps } from './Separator.types';
 
+// Needs to be 1.1px since it sometimes seems to be rounded down to 0.99px which makes it invisible
 const StyledSeparator = styled.div<StyledProps>`
-  width: ${p => (p.vertical ? '1px' : '100%')};
-  height: ${p => (p.vertical ? '100%' : '1px')};
+  width: ${p => (p.vertical ? '1.1px' : '100%')};
+  height: ${p => (p.vertical ? '100%' : '1.1px')};
```
Comment (shmidt-i) [inline src/Atoms/Separator/Separator.tsx]: I reeeally don't like this fix, can we identify which situation cause 0.99px?

File: src/Atoms/Icon/components/SocialYouTube.tsx (hunk 1 above)
Comment (robineng) [reply src/Atoms/Icon/components/SocialYouTube.tsx]: I didn't use svgo. Removed it now

File: src/Atoms/Separator/Separator.tsx (hunk 2 above)
Comment (robineng) [reply src/Atoms/Separator/Separator.tsx]: I reverted the changes and tested it linked to next. It worked so it seems to only be an issue in storybook.

**Pull request merged**